ios/keyman/Keyman/Keyman/GetStartedViewController/GetStartedViewController.swift

```diff
@@ -0,0 +1,195 @@
+  func tableView(_ tableView: UITableView,
+                 cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+    let cellIdentifier = "Cell"
+
+    let reusableCell = tableView.dequeueReusableCell(withIdentifier: cellIdentifier)
+    if let cell = reusableCell {
+      return cell
+    }
+
+    let cell = UITableViewCell(style: .subtitle, reuseIdentifier: cellIdentifier)
+    let selectionColor = UIView()
+    selectionColor.backgroundColor = UIColor(red: 95.0 / 255.0, green: 196.0 / 255.0, blue: 217.0 / 255.0, alpha: 1.0)
+    cell.selectedBackgroundView = selectionColor
+    cell.textLabel?.font = cell.textLabel?.font?.withSize(12.0)
+    cell.detailTextLabel?.font = cell.detailTextLabel?.font?.withSize(10.0)
+    return cell
+  }
+
+  // MARK: - Table view delegate
+  func tableView(_ tableView: UITableView,
+                 willDisplay cell: UITableViewCell,
+                 forRowAt indexPath: IndexPath) {
+    switch indexPath.section {
+    case 0:
+      cell.textLabel?.text = "Add a keyboard for your language"
+      cell.detailTextLabel?.text = ""
+      if !didAddKeyboard() {
+        cell.accessoryType = .none
+      } else {
+        cell.accessoryType = .checkmark
+      }
+    case 1:
+      cell.textLabel?.text = "Set up Keyman as system-wide keyboard"
+      cell.detailTextLabel?.text = ""
+      if !AppDelegate.isKeymanEnabledSystemWide() {
+        cell.accessoryType = .none
+      } else {
+        cell.accessoryType = .checkmark
+      }
+    case 2:
+      cell.textLabel?.text = "More info"
+      cell.detailTextLabel?.text = ""
+      cell.accessoryType = .detailButton
+    default:
+      break
+    }
+  }
+
+  // In a xib-based application, navigation from a table can be handled in -tableView:didSelectRowAtIndexPath:
+  func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+    tableView.cellForRow(at: indexPath)?.isSelected = false
+    performAction(for: indexPath)
+  }
+
+  func tableView(_ tableView: UITableView, accessoryButtonTappedForRowWith indexPath: IndexPath) {
+    performAction(for: indexPath)
+  }
+
+  func performAction(for indexPath: IndexPath) {
+    switch indexPath.section {
+    case 0:
+      mainViewController?.dismissGetStartedView(nil)
+      KMManager.sharedInstance().showKeyboardPicker(in: mainViewController, shouldAddKeyboard: true)
+    case 1:
+      mainViewController?.dismissGetStartedView(nil)
+      let setUpVC = SetUpViewController()
+      mainViewController?.present(setUpVC, animated: true, completion: nil)
+    case 2:
+      mainViewController?.dismissGetStartedView(nil)
+      mainViewController?.infoButtonClick(nil)
+    default:
+      break
+    }
+  }
+
+  func refreshTable() {
+    tableView.reloadData()
+  }
+
+  func didAddKeyboard() -> Bool {
+    let userData = AppDelegate.activeUserDefaults()
+    let userKbs = userData.object(forKey: kKeymanUserKeyboardsListKey) as? [[AnyHashable: String]]
+    guard let userKeyboards = userKbs else {
```

Comment:
OK, some fun stuff about the `guard` statement there:   https://ericcerney.com/swift-guard-statement/

Helps to see why it was added here.